kubernetes-examples/src/main/java/io/fabric8/kubernetes/examples/JobExample.java

```diff
@@ -62,12 +62,16 @@ public static void main(String[] args) {
             .endSpec()
             .build();
 
-          logger.info("Creating job pi.");
+          logger.info("Creating job pi.");                             
           client.batch().v1().jobs().inNamespace(namespace).createOrReplace(job);
 
+          //If you are using version kubernetes-client version 6 or above, use the following code to create job
+          //client.batch().v1().jobs().inNamespace("default").resource(job).create();
+
```

Comment:
Please replace the affected code with valid code instead of adding it as a comment.
The examples should be aligned with whatever version they're published along with (i.e. these examples should be aligned with v6)

Reply:
@Tariqs as long as the job object has the appropriate apiVersion set, then you can use:

client.resource(job).inNamespace("default").create()

Reply:
@shawkins What that apiVersion? Please specify

Reply:
@Tariqs as long as the resource has the appropriate apiVersion and kind (it will by default), you can just use client.resource(object) to perform any of the common create/update/delete operations.  

You only need to call the typed api - batch().v1().jobs() - to call type specific operations, or to get a typed list.

In this example there are other Job specific operations, like getLog, being performed so it would make sense to use:

```
ScalableResource<job> jobResource = client.batch().v1().jobs().inNamespace("default").resource(job);
jobResource.createOrReplace();
...
String jobLog = jobResource.getLog();
```

Reply:
@shawkins When I tried to use it, IntelliJ showed me as deprecated method. I did call like you have mentioned `batch().v1().jobs() `.

Reply:
@shawkins Please refer to this post where I have explained the deprecation issue https://stackoverflow.com/questions/74148689/kubernetes-job-creation-method-showing-as-deprecated-in-intellij-i-am-using-fab/74153050#74153050